skrub/_reporting/_table_report.py

```diff
@@ -143,6 +144,25 @@ def __init__(
         self.max_association_columns = max_association_columns
         self.n_columns = sbd.shape(self.dataframe)[1]
 
+    def _set_minimal_mode(self):
+        """Put the report in minimal mode.
+
+        This is meant to be called by other skrub functions, such as the
+        expressions  ``__repr__``.
+
+        In the minimal mode, the associations and distributions tabs are not
+        shown and the plots and associations are not computed.
+
+        Once set this cannot be undone.
+        """
+        try:
```

Comment:
Maybe add a comment to clarify what this is doing? 

```suggestion
        try: # deleting cached _summary to generate minimal version
```